examples/imagetype/basic/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.0, < 1.6"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.11"
+    }
+  }
+}
```

Comment:
This is not unusually restrictive. We are testing on v1.5.7 and at version v1.6+ Hashicorp decided to change their licensing so that we would have to change ours use continue using it. Restricting the terraform version to the latest minor that is open source makes sense.